### ISSUE TITLE
Add identifier to yaml 

### ISSUE THREAD

**Issue by joostholslag** [SevKohler/OMOCL#8]
I think it would be helpful to add a human readable identifier to the yaml mapping file, of the mapping itself, not just the archetype. The identifier should also be versioned. 
If this spec will (potentially) be part of the openEHR specification, I suggest adopting the openEHR resource model https://specifications.openehr.org/releases/BASE/latest/resource.html#_authored_resource_class. 
Alternatively you could adopt the FHIR canonical resource, which will be more familiar to people familiar with fhir. https://build.fhir.org/canonicalresource.html#CanonicalResource

**Comment by SevKohler** [comment]
@yampeku what do you think about that. 
I would say with the archetype + the version of the mapping provided with the new FHIRconnect+OMOCL header its maybe enough as an identifier ?

Problem will be more colliding mappings for the same archetype.

**Comment by yampeku** [comment]
I can see that being useful, as maybe several mappings could coexist in the future (e.g. conditions coming from different sources that you want to specify types)

**Comment by joostholslag** [comment]
Yes. And you want some name spacing. There will be many mappings for the same archetype. The ckm one should be authoritive. But how will you know which one is from ckm.
